src/utils/apiClient.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+const apiClient = axios.create({
+  baseURL: 'https://frumhire-e18655fb99f3.herokuapp.com/api/', // Base API URL
```

Comment:
i think it could be made more general where you pass a url into it as a param so everyone can use it.

Reply:
i fixed it
it's ok?